content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -154,76 +191,93 @@ ticket.
+ - :guilabel:`Tags`: click into this field to assign an existing tag or create a new one. Multiple
+   tags can be assigned.
 
-Choose an :guilabel:`Invoice Method` from the drop-down. Select :guilabel:`Before Repair` or
-:guilabel:`After Repair` to generate an invoice before or after the work is completed. Selecting
-:guilabel:`No Invoice` means that an invoice cannot be generated for this service.
+If parts are required for the repair, they can be added in the :guilabel:`Parts` tab. Additional
+information for the internal repair team can be added to the :guilabel:`Repair Notes` tab.
 
-If parts are required for the repair, they can be added in the :guilabel:`Parts` tab. Services can
-be added as product lines on the :guilabel:`Operations` tab. Additional information for the internal
-repair team can be added to the :guilabel:`Repair Notes` tab. Information for the customer can be
-added to the :guilabel:`Quotation Notes` tab, and will be automatically added to the PDF of the
-quotations generated from this :guilabel:`Repair Reference`.
+Once the form is complete, click :guilabel:`Confirm Repair`. To create, edit, and send a quote for
+this repair, click :guilabel:`Create Quotation`.
 
-A :guilabel:`Repairs` smart button will be added to the ticket, linking to the repair order.
+A :guilabel:`Repairs` smart button is added to the ticket, linking to the repair order.
 
 .. image:: after_sales/after-sales-repair-smart-button.png
    :align: center
    :alt: View of smart buttons focusing on repair button.
 
-.. note::
-   Once a user creates a repair order from a *Helpdesk* ticket, they will be able to access it
-   through the ticket's :guilabel:`Repair` smart button, or from a link in the :guilabel:`Chatter`,
-   even if they do not have access rights to the *Repair* application.
+.. tip::
+   Once a user creates a repair order from a *Helpdesk* ticket, they can access it through the
+   ticket's :guilabel:`Repair` smart button, or from a link in the chatter, even if they do not have
+   access rights to the *Repair* application.
 
 Create a field service task from a ticket
 =========================================
 
 On-site interventions can be planned from a ticket and managed through the *Field Service*
-application. Customers with :doc:`portal access </applications/general/users/portal>` will be able
-to track the progress of a :guilabel:`Field Service` task just as they would a *Helpdesk* ticket.
+application. Customers with :doc:`portal access </applications/general/users/portal>` are able to
+track the progress of a *Field Service* task just as they would a *Helpdesk* ticket.
+
+.. tip::
+   To change the default *Field Service* project for the team, go to :menuselection:`Helpdesk app
+   --> Configuration --> Helpdesk Teams` to select a :guilabel:`Team`. Scroll to the
+   :guilabel:`After-Sales` section and choose new project under :guilabel:`Field Service`.
```

Comment:
```suggestion
   :guilabel:`After-Sales` section and choose a project under :guilabel:`Field Service`.
```

I only see Field Service - I assume that is the default one, and the one that is recommended to be used? If not, you can expand this part.